server/routes/incentiveLevels.ts

```diff
@@ -17,6 +17,9 @@ export const manageIncentiveLevelsRole = 'ROLE_MAINTAIN_INCENTIVE_LEVELS'
 export default function routes(router: Router): Router {
   const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
 
+  /*
```

Comment:
Does this need to be `/**` to make JSDoc happy? Not that it matters too much tbh

Reply:
there's not always a sensible variable or function to attach it to, so i just used multi-line comments.

my plan was to actually move all this junk into several separate functions and then do better jsdoc-style comments. didn't wanna do in this commit because the diff will be crazy.